specification/columns/contractedcost.md

```diff
@@ -2,7 +2,11 @@
 
 Contracted Cost represents the cost calculated by multiplying [*contracted unit price*](#glossary:contracted-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). Contracted Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on negotiation activities, by comparing it with [List Cost](#listcost). If [*negotiated discounts*](#glossary:negotiated-discount) are not applicable, the Contracted Cost defaults to the List Cost.
 
-The ContractedCost column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. When [ContractedUnitPrice](#contractedunitprice) is present and not null, multiplying the ContractedUnitPrice by PricingQuantity MUST produce the ContractedCost, except in cases of [ChargeClass](#chargeclass) "Correction", which may address PricingQuantity or any cost discrepancies independently.
+The ContractedCost column adheres to the following requirements:
+
+* The ContractedCost column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null.
+* This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency.
+* When [ContractedUnitPrice](#contractedunitprice) is present and not null, multiplying the ContractedUnitPrice by PricingQuantity MUST produce the ContractedCost, except in cases of [ChargeClass](#chargeclass) "Correction", which may address PricingQuantity or any cost discrepancies independently.
```

Comment:
nit: Should we be consistent?
```suggestion
* The ContractedCost column MUST be equal to ContractedUnitPrice multiplied by PricingQuantity when [ContractedUnitPrice](#contractedunitprice) is present and not null, except in cases of [ChargeClass](#chargeclass) "Correction", which may address PricingQuantity or any cost discrepancies independently.
```
Or perhaps separate the condition from the requirement to _maybe_ make it more readable? 🤔 (This removes the PricingQuantity reference since I'm not sure how that adds value here. We can add that back if it's important.)
```suggestion
* When [ContractedUnitPrice](#contractedunitprice) is present and not null, and [ChargeClass](#chargeclass) is not "Correction":
  * The ContractedCost column MUST be equal to ContractedUnitPrice multiplied by PricingQuantity.
```